packages/clap-host/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgname=clap-host
```

Comment:
I think we should talk with the Arch maintainers about that package name.

Reply:
@dvzrv for context: this package installs a `clap-host` binary and our concern is that since other hosts have `provides=(clap-host)` this may conflict with `provides` either being a notion of compatibility (package provides a certain shared lib or binary) or of an abstract service implementation (package supports a plugin format).

An idea would be to rename that to `clap-example-host` now through a patch.

Reply:
Yeah, ideally name the pkg clap-example-host. Pacman unfortunately has a (not very obvious) preference for "real packages" over "virtual provides" (a package named "clap-host" would be auto-chosen over any of the virtual provides).

Reply:
Done in 57b9a889f109b48b6f24b08ece92b3c95b873987